lib/generators/ember-cli/heroku/heroku_generator.rb

```diff
@@ -0,0 +1,21 @@
+module EmberCLI
+  class HerokuGenerator < Rails::Generators::Base
+    source_root File.expand_path("../templates", __FILE__)
+
+    namespace "ember-cli:heroku"
+
+    gem "rails_12factor", group: [:staging, :production]
```

Comment:
This is causing a strange error:

```
/Users/seanpdoyle/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler/rubygems_integration.rb:292:in `bloc
k in replace_gem': rails_12factor is not part of the bundle. Add it to Gemfile. (Gem::LoadError)
```

[gem](http://api.rubyonrails.org/classes/Rails/Generators/Actions.html#method-i-gem) is a generator method, but this looks like it's using the version that executes in the `Gemfile`